qlib/kernel/Kernel_cc.rs

```diff
@@ -1341,6 +1342,9 @@ impl HostSpace {
     }
 
     pub fn Panic_cc(str: &str) {
+        if !LOG_AVAILABLE.load(Ordering::Acquire){
```

Comment:
why is this the case?

Reply:
IO is intercepted in sev_snp, we can do io correctly only after GHCB is initialized. I use the value LOG_AVAILABLE to refer to that.